app/routes/team/index.js

```diff
@@ -35,6 +35,12 @@ export default AuthenticatedRoute.extend(styleBody, CurrentUserSettings, Paginat
                 });
             }
 
+            // we need to load the roles into ember cache
+            // invites return role_id only and we do not offer a /role/:id endpoint
+            modelPromises.roles = this.get('store').query('role', {}).then((roles) => {
```

Comment:
Doesn't matter in this case but for future reference `this.get('store').query('role', {})` would [normally be written](https://guides.emberjs.com/v2.9.0/models/finding-records/#toc_retrieving-multiple-records) as `this.get('store').findAll('role')`